app/assets/javascripts/backbone/views/user_groups_modal_view.js.coffee

```diff
@@ -0,0 +1,12 @@
+class ELMO.Views.UserGroupsModalView extends Backbone.View
+  el: '#user-groups-modal'
+
+  initialize: (params) ->
+    this.params = params
+    this.set_body(params.html)
+
+  set_body: (html) ->
+    # console.log(html)
+    @modal_body = $(@el).find('.modal-body')
+    console.log(@modal_body)
+    $('.modal-body').html(html)
```

Comment:
Should you be using the @modal_body var here?

All this could really be one line: `@$el.find('.modal-body').html(html)`